Automattic/jetpack
# VideoPress: Add raw caption on jetpack_videopress field

Relates to #26341.

#### Changes proposed in this Pull Request:

* Add `caption` to `jetpack_videopress` field returned on the `wp/v2/media` endpoint
* Fetch the raw caption information from the attachment post `post_excerpt` field, as per pe4Cmx-fL-p2#comment-153
* Handle WPCOM sites using the right `$blog_id` while fetching the attachment post

#### Other information:


#### Jetpack product discussion



N/A.

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






* Test on a site with VideoPress plugin installed
* Upload a video
* Pick the attachment id for the uploaded video
* Go to `https://wordpress.com/media/yoursite.com` to edit the video details, setting a caption value for the video

![videopress-caption-field-on-wpcom-edit-media-page](https://user-images.githubusercontent.com/6760046/192061863-c38b987e-9da6-44e5-bb46-5724bd1d0197.png)

* Do a `GET` request on the `/wp/v2/media/{ id }` endpoint
   * Example: `curl -X GET https://yoursite.com/wp-json/wp/v2/media/{ id } | jq`
* Confirm that you see the `caption` in `jetpack_videopress`
* Confirm that the value for the field is the same you set above
* Confirm that the value for the field is similiar to the `caption.rendered`, but without surrounding `<p>s`

![videopress-raw-caption-field-present](https://user-images.githubusercontent.com/6760046/192060498-095b9ed7-48bd-4c5a-93d5-6357b10c95d6.png)

**Note:** this was not tested on WPCOM yet.
